plugin/autoswap.vim

```diff
@@ -109,6 +117,12 @@ function! AS_DetectActiveWindow (filename)
 	return active_window
 endfunction
 
+" TMUX: Detection function for tmux, uses tmux
+function! AS_DetectActiveWindow_Tmux (swapname)
+	let find_win_cmd = 'tmux list-panes -aF "#{pane_tty} #{window_index} #{pane_index}"| grep $(ps h $(fuser '.a:swapname.' 2>/dev/null | grep -o "[0-9]*" || echo "^$") 2>/dev/null | sed "s/^ *//" | cut -d" " -f2) 2>/dev/null'
```

Comment:
Hi, thank you for the pull request, I do not use vim in tmux but it looks very helpful for those who do.

I have one question and one request.

The question: What if the correct tmux is running in another terminal? In theory one should first raise the correct terminal, them switch to the correct pane. Or is this a too uncommon case?

The request: could you please split this long line in separate variables, one for each subshell substitution? For example, the subexpression

    $(fuser '.a:swapname.' 2>/dev/null | grep -o "[0-9]*" || echo "^$")

could be saved in a `pid` variable and the enclosing command changed to something like

    '... ps h ' . pid . '  2>/dev/null | sed ...'